src/playwright/playwright.config.ts

```diff
@@ -82,7 +117,38 @@ export default defineConfig({
         ),
       },
     ],
+    [
+      './coverage-reporter.ts',
+      {
+        include: [
+          // Migrated
```

Comment:
When migrating a component, just search for "migrated" in your editor and add the component to every location you find.